### ISSUE TITLE
Error in Unicorn sync for Foundation.Handlebars

### ISSUE THREAD

**Issue by alphonso77** [mtelligent/Handlebars-For-Sitecore#2]
Two .yml files are named differently in the file system than what .config specifies.:

1) Handlebar Facet Container Sett.yml

https://github.com/mtelligent/Handlebars-For-Sitecore/blob/master/src/Foundation/Handlebars/serialization/ExpBut1/Handlebar%20Facet%20Container%20Sett.yml

2) Handlebar Json Container Setti.yml

https://github.com/mtelligent/Handlebars-For-Sitecore/blob/master/src/Foundation/Handlebars/serialization/ExpBut2/Handlebar%20Json%20Container%20Setti.yml

https://github.com/mtelligent/Handlebars-For-Sitecore/blob/master/src/Foundation/Handlebars/code/Foundation.Handlebars/App_Config/Include/Foundation.Handlebars/Serialization.config

specifies:
`<include name="ExpBut1" database="core" path="/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Handlebar Facet Container Settings"/>
            <include name="ExpBut2" database="core" path="/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Handlebar Json Container Settings"/>`

**Comment by mtelligent** [comment]
Hmm, Is this causing issues in deserializing from Unicorn? Are you getting a warning?

**Comment by alphonso77** [comment]
I received an error when I initially attempted to do a Sync on Foundation.Handlebars. The sync mostly succeeded but reported that those to files were not found and were skipped. When I navigated to that location in the tree in the content editor I could see that they were missing. When I changed their names to match what's in .config the sync operation completed successfully for those two files. I'm sorry, I deleted the exact text of the Unicorn error, but I can probably reproduce it if you desire.

**Comment by mtelligent** [comment]
Thanks for that detail. 

I have a new workstation so needed to clone and configure my environment to see if I could reproduce. After setting everything up, those items synced fine for me. 

From my initial sync of the project:
[A] core:/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Handlebar Facet Container Settings (088ff6b1-9b2c-461e-8cfd-ad4ffc5ae80b)
[A] core:/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Handlebar Json Container Settings (0f5b6f96-b52c-4ea0-a0b1-b68f38bcd0c1)
[A] core:/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Handlebar Query Settings (2b0965a5-c75c-4a01-b358-5fdeb1db9569)
[A] core:/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Handlebar Settings (10f5421c-0028-4f47-8fa8-c5064950e697)
[A] core:/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Edit Template (6d099084-861e-441a-8083-9aa9d956d38c)
[A] core:/sitecore/content/Applications/WebEdit/Custom Experience Buttons/Handlebar Form Settings (1788c4ba-ea0c-4671-94e6-f2d91d89e4a7)

I did end up updating all my nuget feeds and removing the "Facet" project, which was an early version of the "User Settings" project. I replaced that with a dll reference in the lib folder. Let me know if you have any further issues.
